pkg/generator/ingress_translator.go

```diff
@@ -63,6 +64,16 @@ type IngressTranslator struct {
 	tracker         tracker.Interface
 }
 
+type DropRouteConfig struct {
+	// Path that will be dropped
+	Path string `json:"path"`
+}
+
+type DropRoutes struct {
+	// Configurations of routes that will be dropped
+	Routes []*DropRouteConfig `json:"routes"`
```

Comment:
I can't find puroposes of why DropRouteConfig is a pointer, imo should be a value

Reply:
it's faster to have array of pointer than having an array of sturct